properties.schema

```diff
@@ -19,20 +19,12 @@
                 "_isEnabled": {
                   "type":"boolean",
                   "required":true,
+                  "default": true,
                   "title":"Is Enabled",
                   "inputType": {"type": "Boolean", "options": [true, false]},
                   "validators": [],
                   "help": "If set to 'true', this course will be tracked by TinCan/xAPI"
                 },
-                "_activityID": {
```

Comment:
yes, but I suspect if we were to use a zip upload to the LMS (which isn't possible with the [tincanmodlaunch](https://github.com/garemoko/moodle-mod_tincanlaunch) plugin) then the activity id would be defined in the zip, which is generated by grunt-build and uses the value defined in _activityId.